packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -0,0 +1,63 @@
+import { LightningElement, api } from 'lwc';
+import { registerComponentForInit, initializeWithHeadless } from 'c/quanticHeadlessLoader';
+
+
+export default class QuanticTab extends LightningElement {
+  /** @type {import("coveo").Tab} */
+  tab;
+  /** @type {()=> void} */
+  unsubscribe;
+  /** @type {boolean} */
+  isActive = false;
+  /** @type {string} */
+  @api label;
+  /** @type {string} */
+  @api expression = "";
+  /** @type {string} */
+  @api engineId;
+
+
+  connectedCallback() {
+    registerComponentForInit(this, this.engineId);
+  }
+
+  renderedCallback() {
+    initializeWithHeadless(this, this.engineId, this.initialize.bind(this));
+  }
+
+  /**
+  * @param {import("coveo").SearchEngine} engine
+  */
+  @api
+  initialize(engine) {
+    this.tab = CoveoHeadless.buildTab(engine, {
+      options: {
+        expression: this.expression,
+        id: this.label,
+      },
+      initialState: {
+        isActive: this.isActive,
+      }
+    });
+    this.unsubscribe = this.tab.subscribe(() => this.updateState());
+  }
+
+  disconnectedCallback() {
+    if (this.unsubscribe) {
+      this.unsubscribe();
+    }
+  }
+
+  updateState() {
+    this.isActive = this.tab.state.isActive;
+  }
+
+  select() {
+    this.tab.select();
+  }
+
+  get activeTabClass() {
```

Comment:
```suggestion
  get tabClass() {
```

The tab is not always active.